File: Data Structures & Algorithms/length-of-last-word/submission-2.py
class Solution:
    def lengthOfLastWord(self, s: str) -> int:
        r = len(s) - 1;
        while (s[r] == " "):
            r -= 1;  

        count = 0
        while (r >= 0 and s[r] != " "):
            count += 1
            r -= 1
        return count    

File: Data Structures & Algorithms/max-consecutive-ones/submission-2.js
class Solution {
    /**
     * @param {number[]} nums
     * @return {number}
     */
    findMaxConsecutiveOnes(nums) {
        let countM = 0;
        let count = 0;

        for (const num of nums){
            if(num === 1) {
                count ++;
                countM = Math.max(countM, count)
            }
            else{
                count = 0
            }
        }
        return countM;

    }
}
